commons/src/lib.rs

```diff
@@ -25,9 +25,21 @@ pub mod prelude_errors {
 }
 
 use actix_web::http::{header, HeaderMap};
+use std::collections::HashMap;
 use std::collections::HashSet;
 use url::form_urlencoded;
 
+lazy_static! {
+    /// list of cincinnati versions
+    pub static ref CINCINNATI_VERSION: HashMap<&'static str, i32> =
+        [("application/vnd.redhat.cincinnati.v1+json", 1)]
+            .iter()
+            .cloned()
+            .collect();
+    /// minimum cincinnati version supported
+    pub static ref MIN_CINCINNATI_VERSION: &'static str = "application/vnd.redhat.cincinnati.v1+json";
```

Comment:
Perhaps we want `default` here instead? `min` should be decided from `CINCINNATI_VERSION` hashmap?